src/API/Site/Controllers/Ads/BudgetRecommendationController.php

```diff
@@ -68,31 +68,60 @@ public function register_routes(): void {
 	 */
 	protected function get_budget_recommendation_callback(): callable {
 		return function( Request $request ) {
-			$country        = strtoupper( $request->get_param( 'country_code' ) );
-			$currency       = $this->middleware->get_ads_currency();
-			$recommendation = $this
+			$country_codes = $request->get_query_params()['country_codes'] ?? '';
```

Comment:
Thanks for the info. Also updated in eb0425b to use a new schema for query parameters, and to use `get_params` instead of `get_query_params` to get the `country_codes` parameter.